### DISCUSSION TITLE
Reviewing notes without flashcards?

### DISCUSSION THREAD

**Discussion by swar8080** [st3v3nmw/obsidian-spaced-repetition#806] [Q&A] [answered]
Hello! I'm looking to review entire notes with spaced repetition without using the flashcard feature. Is this possible?

I've tagged some notes using _Settings > Tags to review_, but the status bar always shows 0 notes to review. Other than that, all the settings are the defaults and none of the notes have flashcards defined

Thank you

**Comment by ronzulu** [comment, marked as answer]
Hi @swar8080 

I'm wondering what shows up in the "note review pane"

For that panel to show, make sure that the following option is enabled:
![image](https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/75528127/bfcc79ff-2198-461d-8da5-a6e66ad25ce8)

I don't use the note review feature much, but this is what I see:
![image](https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/75528127/bf4b7223-dcad-44f4-a6c2-a07c0e84413d)

Cheers
Ronny

**Reply by swar8080** [reply]
Ah, I completely missed the review pane. It seems to do what i'm looking for

Thanks!

**Comment by ronzulu** [comment]
No worries, I missed it a couple of months ago too

The incorrect value for the count of notes to review sounds similar to this outstanding bug https://github.com/st3v3nmw/obsidian-spaced-repetition/issues/437

Cheers
Ronny
